### ISSUE TITLE
curlopt_stderr is reset every time hhb_curl_exec2 is called, and on some systems, it complains that NULL is  not a valid handle..

### ISSUE THREAD

**Issue by divinity76** [divinity76/hhb_.inc.php#6]

**Comment by divinity76** [comment]
for instance, php 5.4 CLI *but only CLI, not when called from php5-fpm / apache mod php5...

``` php
<?php 
error_reporting(E_ALL);
ini_set('display_errors','1');
$ch=curl_init();
$ret=curl_setopt($ch,CURLOPT_STDERR,NULL);
var_dump($ret);
```

Warning: curl_setopt(): supplied argument is not a valid File-Handle resource in /home1/hgtest1/public_html/curlbugtest.php on line 5
bool(false)

**Comment by divinity76** [comment]
this was fixed in commit 629541ac1ee23942235d6ac10e20f6a3327f6fe8
